tests/test_user.py

```diff
@@ -0,0 +1,12 @@
+from tests.base_test import BaseTest
+
+new_user = {
```

Comment:
You could consider having the dummy data in the `__init__` file for the tests since they will be used across all the tests files you could have.

Reply:
Ok thank you for that review, let me find a way of implementing it as it seems a better way of minimising time wastage